### DISCUSSION TITLE
goreleaser with go-semantic-release ?

### DISCUSSION THREAD

**Discussion by iangregsondev** [goreleaser/goreleaser#1626] [Q&A] [answered]
Hi there,

I was wondering if anybody is using the workflow of

1. go-semantic-release (standalone version of semantic-release)
2. goreleaser

Not to sure if these 2 can be used together but this was my thinking (I am already using semantic release for frontend projects).

go-semantic-release supports conventional commits - so it can be used to decide on the version and "TAG"

goreleaser then would run and pickup the newly created tag.

goreleaser would be responsible for publishing docker images etc.

I would really love to hear if anybody is using this flow or a similar flow.

Thanks in advance

**Comment by caarlos0** [comment, marked as answer]
never used `go-semantic-release`.

I do use one small tool I crafted years ago (https://github.com/caarlos0/svu) which only informs me of what the next tag should be...

so my workflow is something like:

```sh
git tag `svu next`
git push --tags
```

and wait for the CI to run...

**Reply by anbraten** [reply]
Would be nice if that could be somehow integrated to goreleaser, so goreleaser could be executed on push to master / main.

**Reply by caarlos0** [reply]
pushing tags is out of the goreleaser scope.

**Comment by grzesuav** [comment]
@iangregsondev https://github.com/go-semantic-release/hooks-goreleaser seems like you can use this
